Sources/CorePayments/Networking/AnalyticsService.swift

```diff
@@ -45,7 +45,7 @@ public struct AnalyticsService {
             let clientID = coreConfig.clientID
             
             let eventData = AnalyticsEventData(
-                environment: http.coreConfig.environment.toString,
```

Comment:
Can we remove `toString` from the `Environment` enum all together? Looks like this is the only place it was used.